LetsDoStuff.WebApi/Controllers/ParticipationRequestorController.cs

```diff
@@ -9,14 +9,33 @@
 
 namespace LetsDoStuff.WebApi.Controllers
 {
-    [Route("api/participation")]
-    public class ParticipationController : ControllerBase
+    [Route("api/ParticipationRequestor")]
```

Comment:
Why do you change the route?

Reply:
Because now we have two routes for participation: the one for an ordinary user and the second for a creator. In the beginning, they both had similar names: ParticipationRequestor and ParticipationAcceptor. But by now they have names like ParticipationRequestor and ParticipantAcceptor. I wanted to show that these two controllers have a similar job but different goals.

Reply:
it is better to operate business objects in EP naming

Reply:
Sorry, I don't get it. Do you prefer I should rename these routes something like participation and participant? Or should I merge these two controllers like cervices ( that we discussed below ) as well? EP naming?!